### PR TITLE
updates project to compile with dynamic framework

### PR DESCRIPTION
**Pull request by esreli**

### CODE REVIEW

**Comment by nixta** [comment]
Does this add the strip frameworks build phase? https://developers.arcgis.com/ios/latest/swift/guide/install.htm#ESRI_SECTION3_349055405B2D4F74A57F902C65C4F197

File: maps-app-ios.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -741,6 +756,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
```
Comment (philium) [inline maps-app-ios.xcodeproj/project.pbxproj]: `English` is a deprecated identifier, so I would suggest changing `developmentRegion` to `en` instead.

Comment (nixta) [reply maps-app-ios.xcodeproj/project.pbxproj]: Interesting. Is that a relatively recent deprecation?

Comment (philium) [reply maps-app-ios.xcodeproj/project.pbxproj]: @nixta I'm not sure, but the warning about it being deprecated appeared in Xcode 10.2.

**Pull request merged**